tests/wallet/encryptDecryptMnemonic.test.ts

```diff
@@ -1,6 +1,6 @@
 import { beforeEach, describe, expect, it, vi } from 'vitest';
 
-import { decryptMnemonicWithCallback, encryptMnemonicWithCallback } from '../../wallet';
+import { decryptMnemonicWithCallback, encryptMnemonicWithCallback } from '../../seedVault/encryptionUtils';
```

Comment:
This file should probably move to the seedvault dir since it's only testing that and nothing specific to the wallet.